index.html

```diff
@@ -25,32 +25,33 @@
 <body>
     <div class="page-container flex small_bg">
 
+        
         <header>
             <nav class="flex">
                 <a href="" class="logo">Welcome</a>
-                <a href="" class="menu">
+                <button class="menu">
                     <img src="images/Menu.svg" alt="a Union icon" class="menu-icons">
-                </a>
+                </button>
             </nav>
         </header>
 
         <section id="s-1" class="flex">
             <ul class="flex navlinks">
-                <li>
-                    Portfolio
+                <li class="navlink-items">
+                    <a href="#Works-grid">Portfolio</a>
```

Comment:
In your desktop version, after I click on any nav link, the whole menu disappears, likely your functionality in your mobile version made this new bug in your website, kindly fix it 😉 

- [ ] Your nav bar should be there even if you click any link before.

**Before clicking any navbar link ( in your desktop version)**

![image](https://user-images.githubusercontent.com/60164239/131472143-99af304e-5f30-46f5-a1d0-50eff9c32b0c.png)

**After clicking any navbar link ( in your desktop version)**

![image](https://user-images.githubusercontent.com/60164239/131471942-6bb1d5a4-fc4d-4ecf-8066-b69613ff2a7f.png)

Reply:
* done 👍